src/SwipeableDrawer/SwipeableDrawer.js

```diff
@@ -69,12 +69,8 @@ class SwipeableDrawer extends React.Component {
 
   getTranslate(current) {
     const start = isHorizontal(this.props) ? this.startX : this.startY;
-
     return Math.min(
-      Math.max(
-        this.isSwiping === 'closing' ? start - current : this.getMaxTranslate() + start - current,
-        0,
-      ),
+      Math.max(this.props.open ? start - current : this.getMaxTranslate() + start - current, 0),
```

Comment:
Dropping the dependency on the swiping direction is intentional. It doesn't matter at all here.